book/src/user/startup.md

```diff
@@ -0,0 +1,190 @@
+# Zebra Startup & Sync Process
+
+When Zebra starts up, it launches [a lot of concurrent tasks](https://github.com/ZcashFoundation/zebra/blob/main/zebrad/src/commands/start.rs).
+
+## Grafana Dashboards
+
+You can see what Zebra is doing during the sync using its [Grafana dashboards](metrics.md).
+
+## Example Startup Logs
+
+Zebra logs the progress of each of its major startup tasks.
+
+In this section, we re-ordered some logs, trimmed (`...`) verbose parts of the logs, and re-formatted some long lines.
+
+(The order of the logs or tasks might be different in each Zebra run, because these tasks are run concurrently.)
+
+### Load Configuration
+
+Zebra loads its configuration from its config file.
+
+Some important parts of the config are:
+- `consensus.checkpoint_sync`: use all the checkpoints, or just use the mandatory ones
+- `network.listen_addr`: the Zcash listener port
+- `network.network`: the configured Zcash network (mainnet or testnet)
+- `network.initial_mainnet_peers`: the initial DNS seeder or fixed address peers
+- `network.peerset_initial_target_size`: the initial target size of the peer set, also used to calculate connection limits
+- `state.cache_dir`: where the cached state is stored on disk
+- `rpc.listen_addr`: optional JSON-RPC listener port
+
+See [the full list of configuration options](https://doc.zebra.zfnd.org/zebrad/config/struct.ZebradConfig.html).
+
+```
+zebrad::commands::start: Starting zebrad
+zebrad::commands::start: config=ZebradConfig {
+  consensus: Config { checkpoint_sync: true, ... },
+  network: Config {
+    listen_addr: 127.0.0.1:8233,
+    network: Mainnet,
+    initial_mainnet_peers: {
+      "dnsseed.z.cash:8233",
+      "mainnet.is.yolo.money:8233",
+      "mainnet.seeder.zfnd.org:8233",
+      "dnsseed.str4d.xyz:8233"
+    },
+    peerset_initial_target_size: 25, ...
+  },
+  state: Config { cache_dir: "/home/user/.cache/state/v24/mainnet", ... },
+  rpc: Config { listen_addr: Some(127.0.0.1:8232) }, ...
+}
+```
+
+### Open Cached State
+
+Zebra opens the configured cached state, or creates a new empty state.
+
+```
+zebrad::commands::start: initializing node state
+zebra_state::service::finalized_state::disk_db: the open file limit is high enough for Zebra current_limit=1048576 min_limit=512 ideal_limit=1024
+zebra_state::service::finalized_state::disk_db: Opened Zebra state cache at /home/user/.cache/state/v24/mainnet
+zebra_state::service::finalized_state: loaded Zebra state cache tip=None
+zebra_state::service: created new read-only state service
+...
+```
+
+### Peer Handling
+
+#### Discover Initial Peers
+
+Zebra queries DNS seeders for initial peer IP addresses, then tries to connect to `network.peerset_initial_target_size` initial peers.
```

Comment:
```suggestion
Zebra queries [DNS seeders](https://github.com/ZcashFoundation/dnsseeder) for initial peer IP addresses, then tries to connect to `network.peerset_initial_target_size` initial peers.
```

Reply:
There are 4 different DNS seeders, and only 2 of them run the Foundation's software at that link.

Do you think we could link to a definition instead? Or would it be confusion to use the Bitcoin definition?
https://btcinformation.org/en/glossary/dns-seed